MAUI/Carousel-View/Populating-Data.md

```diff
@@ -0,0 +1,275 @@
+                       Aspect="AspectFit"/>
+            </DataTemplate>
+        </ResourceDictionary>
+    </ContentPage.Resources>
+    <ContentPage.Content>
+        <carousel:SfCarousel x:Name="carousel"  
+                             ItemTemplate="{StaticResource itemTemplate}" 
+                             ItemsSource="{Binding ImageCollection}" 
+                             HeightRequest="400" 
+                             WidthRequest="800" />
+    </ContentPage.Content>
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+using Syncfusion.SfCarousel.XForms;
+using System.Collections.Generic;
+using Xamarin.Forms;
+
+namespace CarouselSample
+{
+    public partial class MainPage : ContentPage
+    {
+        public MainPage()
+        {
+            InitializeComponent();
+            CarouselViewModel carouselViewModel = new CarouselViewModel();
+
+            SfCarousel carousel = new SfCarousel()
+            {
+                HeightRequest = 400,
+                WidthRequest = 800
+            };
+
+            var itemTemplate = new DataTemplate(() =>
+            {
+                var grid = new Grid();
+                var nameLabel = new Image();
+                nameLabel.SetBinding(Image.SourceProperty, "Image");
+                grid.Children.Add(nameLabel);
+                return grid;
+            });
+
+            carousel.ItemTemplate = itemTemplate;
+            carousel.ItemsSource = carouselViewModel.ImageCollection;
+
+            this.Content = carousel;
+        }
+    }
+
+    public class CarouselModel
+    {
+        public CarouselModel(string imageString)
+        {
+            Image = imageString;
+        }
+        private string _image;
+
+        public string Image
+        {
+            get { return _image; }
+            set { _image = value; }
+        }
+    }
+
+    public class CarouselViewModel
+    {
+        public CarouselViewModel()
+        {
+            ImageCollection.Add(new CarouselModel("carousel_person1.png"));
+            ImageCollection.Add(new CarouselModel("carousel_person2.png"));
+            ImageCollection.Add(new CarouselModel("carousel_person3.png"));
+            ImageCollection.Add(new CarouselModel("carousel_person4.png"));
+            ImageCollection.Add(new CarouselModel("carousel_person5.png"));
+        }
+
+        private List<CarouselModel> imageCollection = new List<CarouselModel>();
+        public List<CarouselModel> ImageCollection
+        {
+            get { return imageCollection; }
+            set { imageCollection = value; }
+        }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+* Now set the `BindingContext` for the items collection in code behind.
+
+{% highlight C# %}
+
+	carousel.BindingContext = new CarouselViewModel();
+
+{% endhighlight %}
+
+## Through Carousel Item
+
+Different set of views can be provided to every items through `ItemContent` property available in `SfCarouselItem` class.
```

Comment:
Different views can be provided to every item through the `ItemContent` property in the `SfCarouselItem` class.

Reply:
Addressed the changes.